pkg/config/config.go

```diff
@@ -1586,3 +1592,21 @@ func GetVectorURL(datatype DataType) (string, error) {
 	}
 	return "", nil
 }
+
+// GetInventoriesMinInterval gets the inventories_min_interval value, applying the default if it is zero.
+func GetInventoriesMinInterval() time.Duration {
+	minInterval := time.Duration(Datadog.GetInt("inventories_min_interval")) * time.Second
+	if minInterval == 0 {
+		minInterval = DefaultInventoriesMinInterval
+	}
+	return minInterval
+}
+
+// GetInventoriesMaxInterval gets the inventories_max_interval value, applying the default if it is zero.
+func GetInventoriesMaxInterval() time.Duration {
```

Comment:
What do you think about adding a generic function such as `GetDurationAsSecondOrDefault(key string, defaultValue time.Duration) time.Duration`?